react/GroupOption.tsx

```diff
@@ -26,36 +26,38 @@ const GroupOption: React.FC<OptionProps> = ({
 
   return (
     <Fragment>
-      <button
-        className={classnames(
-          styles.groupOption,
-          className,
-          'w-100 tl pointer db lh-copy c-on-base bg-base hover-bg-action-secondary ph5-ns pv5-ns flex items-center justify-between bn',
-          {
-            pv4: !lean,
-            pv3: lean,
-          }
-        )}
-        onClick={onClick}
-        role="option"
-        aria-selected={selected}
-      >
-        {children}
-        <span
+      <div className={classnames({ 'pl5 pl0-ns': !selected })}>
```

Comment:
Idk, it does not feel right on mobile. Is that correct? Shouldn't this padding be applied all the time?

<img height="500" src="https://user-images.githubusercontent.com/26108090/99303050-0b1dac00-282f-11eb-930a-d95b706f5a71.png" />

Reply:
Not so easy to notice that on the screenshot I guess, but the "Motoboy" label is really close to the screen border

Reply:
it shouldn't be like that, I'll take a look

Reply:
ohhhhh, I see

I think this check is wrong, it will remove the padding of all selected SLAs even if they don't have the selected SLA styling (with the gray background):
<img width="659" alt="image" src="https://user-images.githubusercontent.com/10223856/99305036-fe4e8780-2831-11eb-88cd-0b3ed323e8df.png">

Reply:
ohh that makes sense. gj

Reply:
I just force-pushed the changes, hopefully its fixed